azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java

```diff
@@ -156,6 +156,12 @@ public void updateTrigger(Trigger t) throws TriggerManagerException {
     synchronized (syncObj) {
       runnerThread.deleteTrigger(triggerIdMap.get(t.getTriggerId()));
       runnerThread.addTrigger(t);
+
+      try {
+        triggerLoader.updateTrigger(t);
+      } catch (TriggerLoaderException e) {
+        throw new TriggerManagerException(e);
```

Comment:
Hi @HappyRay , thanks for your suggestion. It's still not clear to me how to implement your thoughts.